lab-keiran/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+var testFiles = ['test/**/*.js'];
+var appFiles = ['lib/**/*.js', 'bin/cowsayCli.js'];
+gulp.task('lint:app', () => {
+  gulp.src(appFiles)
+    .pipe(eslint({
+      rules: {
+        'indent': [2,2]
+      },
+      envs: [
+        'node',
+        'es6'
+      ]
+    }))
+    .pipe(eslint.format());
+});
+
+gulp.task('lint:test', () => {
+  gulp.src(testFiles)
+    .pipe(eslint({
+      rules: {
+        'indent': [2,2]
+      },
+      envs: [
+        'node',
+        'es6',
+        'mocha'
+      ]
+    }))
+    .pipe(eslint.format());
+});
+
+gulp.task('default', ['lint:app', 'lint:test']);
```

Comment:
Nice! I see that you moved some of the eslintrc options into your gulpfile. However, you're missing a task that will run your mocha tests. Please include it and resubmit, thanks!

Also, even though you moved some of the eslintrc options here, you still need to make sure to include the class .eslintrc file in your project folder in order for Travis to run the linting properly (and also so the TAs can check the linting).